code_schemes/s01e01.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
These should probably be in a different PR, as we should be asking for a review from RDA as well

Reply:
This code scheme looks a bit odd - it's only made out of control codes? Is this because we don't have a coding frame yet?

Reply:
Exactly. As mentioned in the PR description and the code translator. Only reason it isn't here is because comments break the JSON.

Reply:
Not requesting a research review because there is no scheme to review - just need enough for testing

Reply:
I'm pondering whether we should move to JSON + comments. Something like a subset of JSON5?

If we restrained ourselves to single line comments, without escapes, it's very easy to strip them out before parsing. What do think?